include/constants/bench_locations.h

```diff
@@ -0,0 +1,8 @@
+#ifndef GUARD_CONSTANTS_BENCH_LOCATIONS
+#define GUARD_CONSTANTS_BENCH_LOCATIONS
+
+#define BENCH_HOME                  1 //test location in Pallet Town
```

Comment:
Do we want to keep the test location? could also cannibalize it for the Viridian bench here or later on